esmvaltool/recipes/recipe_miles_block.yml

```diff
@@ -0,0 +1,81 @@
+###############################################################################
+## namelist_miles.xml
+##
+## Description
+## Namelist for computing blocking using the MiLES package by P. Davini (ISAC-CNR)
+## MiLES (Mid-Latitude Evaluation System) v0.31
+##
+## Authors
+## Paolo Davini (ISAC-CNR, Italy - p.davini@isac.cnr.it)
+## J. von Hardenberg (ISAC-CNR, Italy - j.vonhardenberg@isac.cnr.it)
+## E. Arnone (ISAC-CNR, Italy - e.arnone@isac.cnr.it)
+##
+## Project
+## Copernicus C3S-MAGIC (C3S 34a Lot2) 
+##
+## References:
+## https://github.com/oloapinivad/MiLES
+## Davini, P., C. Cagnazzo, S. Gualdi, and A. Navarra, 2012: Bidimensional Diagnostics, Variability, and Trends of Northern Hemisphere Blocking. J. Climate, 25, 6496–6509, doi: 10.1175/JCLI-D-12-00032.1
+## Tibaldi S, Molteni F. 1990. On the operational predictability of blocking. Tellus A 42(3): 343–365, doi:10.1034/j.1600- 0870.1990.t01- 2- 00003.x
+##
+## This namelist is part of the ESMValTool
+################################################################################
+#
+# mask_landocean: false
+# multi_model_statistics: false
+# reference_dataset: "EC-Earth"
+---
+documentation:
+  description: |
+    Recipe for computing and plotting blocking statistics using 
+    the MiLES (Mid-Latitude Evaluation System) package
+
+  authors:
+    - hard_jo
+    - davi_pa
+    - arno_en
+
+  references:
+    - davini18
+    - davini12jclim
+    - tibaldi1990tel
+
+  projects:
+    - c3s-magic
+
+datasets:
+   - {dataset: EC-Earth,    project: CMIP5,  exp: historical,  ensemble: r8i1p1,  start_year: 1980,  end_year: 1989}
```

Comment:
I'm having trouble finding this data. I think the dataset should be `EC-EARTH` instead of `EC-Earth`, but even then I can only find ensemble members `r1i1p1` and `r12i1p1` for variable `zg` with synda. On Jasmin I can find a few more ensemble members, but variable `zg` is only availble in `r1i1p1`, `r2i1p1`, `r7i1p1`, `r9i1p1`, `r12i1p1`.

Reply:
Ok, I changed the recipe to using EC-EARTH and to use r1i1p1.